plone/app/event/portlets/portlet_events.py

```diff
@@ -27,7 +25,16 @@
 
 try:
     from plone.app.contenttypes.behaviors.collection import ISyndicatableCollection as ICollection  # noqa
+    from plone.app.contenttypes.interfaces import IFolder
+    search_base_uid_source = CatalogSource(object_provides={
+        'query': [
+            ICollection.__identifier__,
+            IFolder.__identifier__
+        ],
+        'operator': 'or'
+    })
```

Comment:
this is the same as in portlet_calendar.py - better move it to one place instead of copy paster coding?